README.md

```diff
@@ -1,2 +1,81 @@
 # django-ilmoitin
-Django app for sending notifications
+
+[![Latest PyPI version](https://badge.fury.io/py/django-ilmoitin.svg)](https://pypi.python.org/pypi/django-ilmoitin)
+[![Python versions](https://img.shields.io/pypi/pyversions/django-ilmoitin.svg)](https://pypi.python.org/pypi/django-ilmoitin)
+
+A templated Django messaging library
+
+# Installation
+
+1. `pip install django-ilmoitin`
+
+2. Add `django_ilmoitin` to `INSTALLED_APPS`.
+
+# Usage
+
+1. `django-ilmoitin` uses [`django-mailer`](https://github.com/pinax/django-mailer) to send emails, so you need to configure the `MAILER_EMAIL_BACKEND` setting to let [`django-mailer`](https://github.com/pinax/django-mailer) know, how to actually send the mail:
```

Comment:
I would try to keep the same 88 char line length here too, or at least 120 chars, and split this across a couple of lines, since markdown will display them as one long line anyway, unless you separate them with an empty line

Reply:
and add an empty line before the following code block, for the readability of this file